app/components/Main.js

```diff
@@ -11,7 +10,7 @@ var Main = React.createClass({
           </div>
         </nav>
         <div className="container">
-          <RouteHandler />
+          {this.props.children}
```

Comment:
What's nice about this is that we no longer need to have a react-router dependency